examples/js/scatterplot.js

```diff
@@ -754,4 +754,38 @@ function Scatterplot(el, component, svgClass) {
     return str.replace(' ', '').toLowerCase()
   }
 }
+
+  var scatterplot1Tab1 = document.querySelector('.scatterplot-1-tab-1');
```

Comment:
I've updated it to this way, however, upon hovering the third tab the line starts from the first tab and then moves to the third tab. Whereas, in the js implementation the line starts from whatever tab you first hover on.
![scatterplot_tabs_animation](https://cloud.githubusercontent.com/assets/10215203/22339610/332aafe4-e3b0-11e6-8d9d-73b1b3fc7b81.gif)